custom/icds/static/icds/js/custom_sms_report.js

```diff
@@ -1,12 +1,17 @@
 hqDefine("icds/js/custom_sms_report",[
     'jquery',
+    'hqwebapp/js/initial_page_data',
     'hqwebapp/js/daterangepicker.config', //createBootstrap3DefaultDateRangePicker
 ], function (
-    $
+    $,
+    initial_page_data
 ) {
     var $el = $('#date_range_selector');
     var $startDate = $('#report_start_date');
     var $endDate = $('#report_end_date');
+    var $submit_btn = $('#request_report');
+    if(initial_page_data.get('disable_submit'))
+        $submit_btn.attr('disabled', true);
```

Comment:
why not do this at form itself?

Reply:
I tried but did not get any success. So did it with JS.

Reply:
Let me know what you tried, here is an example where its done.
https://github.com/dimagi/commcare-hq/blob/e493c3c22e2802a571055dd34ac12d00cbeaa86d/corehq/apps/accounting/forms.py#L1061

Reply:
I was trying `attrs={'hidden': disable_submit}`, it was always staying disabled as if we add a disabled attribute then whatever it's value is it will be disabled. We need to completely remove the attribute. So I was not able to figure out how to do that. 
Example looks good, I think ` **({'disabled': True} if self.subscription.is_active else {}) ` was what I need. Thanks 😄